src/Microsoft.Extensions.Logging.AzureAppServices/AzureAppServicesLoggerFactoryExtensions.cs

```diff
@@ -53,10 +82,69 @@ public static ILoggerFactory AddAzureWebAppDiagnostics(this ILoggerFactory facto
         /// <param name="settings">The setting object to configure loggers.</param>
         public static ILoggerFactory AddAzureWebAppDiagnostics(this ILoggerFactory factory, AzureAppServicesDiagnosticsSettings settings)
         {
-            if (WebAppContext.Default.IsRunningInAzureWebApp)
+            var context = WebAppContext.Default;
+            if (context.IsRunningInAzureWebApp)
```

Comment:
Nit/personal preference: when a block wraps the entire contents of a method, invert the condition and return early. It reduces the nesting.